server/lib/notifications/notification-service.ts

```diff
@@ -67,6 +71,24 @@ export default class NotificationService {
     this.nydus.publish(getNotificationsPath(notificationProps.userId), addEventData)
   }
 
+  /**
+   * Clears, i.e. makes not visible, all notifications before a given timestamp for a particular
```

Comment:
nit: this extra explanation isn't really necessary, just saying `Clears all notifications before a given timestamp for a particular user.` is fine